src/main/java/com/google/maps/model/OpeningHours.java

```diff
@@ -0,0 +1,58 @@
+package com.google.maps.model;
+
+public class OpeningHours {
+  /**
+   * openNow is a boolean value indicating if the place is open at the current time.
+   */
+  public Boolean openNow;
+
+  static public class Period {
```

Comment:
Could you reuse Joda's concept of a DayTime [Period](http://joda-time.sourceforge.net/apidocs/org/joda/time/Period.html) here?